<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetsResizableHeaderAttachmentShowColumnsWidget</class>
 <widget class="QWidget" name="WidgetsResizableHeaderAttachmentShowColumnsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="placeholderText">
        <string>Search</string>
       </property>
       <property name="clearButtonEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListView" name="listView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
